### PR TITLE
Add test for `shipcat kong`

### PR DESCRIPTION
**Pull request by janslow**
Add some tests for the existing Kong functionality before starting work on #60

### CODE REVIEW

File: shipcat_cli/tests/common.rs [hunk 1]
```diff
@@ -203,3 +203,61 @@ fn templating_test() {
...
+
+use shipcat::kong;
+use shipcat_definitions::structs::kongfig;
+
+#[test]
+fn kong_test() {
+    setup();
+    let (conf, reg) = Config::new(ConfigType::Base, "dev-uk").unwrap();
+    let kongrs = kong::generate_kong_output(&conf, &reg);
+    assert!(kongrs.is_ok());
```
Comment (clux) [inline shipcat_cli/tests/common.rs]: you can just unwrap the `kongrs` in the line above, that gives a nicer trace if it fails.

File: tests/services/fake-ask/shipcat.yml [hunk 2]
```diff
@@ -66,3 +66,5 @@ configs:
 dependencies:
 - name: fake-storage
   intent: "testing graph module"
+kong:
+  uris: /ai-auth
```
Comment (clux) [inline tests/services/fake-ask/shipcat.yml]: newline

File: shipcat_cli/tests/common.rs (hunk 1 above)
Comment (janslow) [reply shipcat_cli/tests/common.rs]: done

File: tests/services/fake-ask/shipcat.yml (hunk 2 above)
Comment (janslow) [reply tests/services/fake-ask/shipcat.yml]: done

**Comment by janslow** [comment]
@clux can you merge, I don't have write access

**Comment by clux** [comment]
i'll give you write access

**Pull request merged**